reactDashboard/src/components/DataDefinitions/DataDefinitions.jsx

```diff
@@ -75,29 +75,64 @@ function DataDefinitions() {
         </Card.Header>
         <Accordion.Collapse eventKey="0">
           <Card.Body className="data-definitions-body">
-            <p>
-              There may be some minor fluctuations in the daily number of cases
-              due to laboratory reporting delays for specimen dates and
-              additional information becoming available, in which case the data
-              is revised in a future next update.{" "}
-              {createLink(
-                "https://www.opendata.nhs.scot/dataset/covid-19-in-scotland",
-                "Click here for further information relating to the accuracy and completeness of the Public Health Scotland Covid-19 dataset."
-              )}
-            </p>
+            <p className="definitions-heading">Reporting and Specimen Dates</p>
+            <hr />
             <ul>
               <li>
                 <span className="defined-term">Reported Dates</span>: Since the
                 time taken to test samples and report the results varies, new
                 cases reported on a daily basis in the headline summary figures
                 on the dashboard pages may be distributed across a range of
-                Specimen Dates.
+                specimen dates.
               </li>
               <li>
                 <span className="defined-term">Specimen Dates</span>:{" "}
                 {specimenDateDefinition()}
               </li>
             </ul>
+            <p>
+              There is a reporting delay in testing results, so data on tests
```

Comment:
Isn't this already in the other PR? Having it in two branches can get messy

Reply:
I will merge the other PR into this one once it is approved and the duplication will be resolved, but made more sense with the changes that were required to start from the same place as the branch that was currently under review